<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkRemoteSettingsPage</class>
 <widget class="QWidget" name="NetworkRemoteSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Network Remote</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="use_remote">
     <property name="text">
      <string>Use Remotecontrol</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="use_remote_container" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="leftMargin">
       <number>24</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_remote_port">
          <property name="minimumSize">
           <size>
            <width>171</width>
            <height>0</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Port</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="remote_port">
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="value">
           <number>8080</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="only_non_public_ip">
          <property name="toolTip">
           <string>Only accept connections from clients within the ip ranges:
10.x.x.x
172.16.0.0 - 172.31.255.255
192.168.x.x</string>
          </property>
          <property name="text">
           <string>Accept non public clients only</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>36</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>use_remote</sender>
   <signal>toggled(bool)</signal>
   <receiver>use_remote_container</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
